src/utils/azureUtils.ts

```diff
@@ -22,18 +25,100 @@ export function getResourceGroupFromId(id: string): string {
     return parseResourceId(id)[2];
 }
 
-export function createGroupConfigFromResource(resource: GenericResource): GroupingConfig {
+export function createGroupConfigFromResource(resource: GenericResource, subscriptionId: string | undefined): GroupingConfig {
```

Comment:
Should it be `AppResource` instead of `GenericResource` here?

Reply:
I think in this case, it should be `GenericResource`.  We rely on some properties that are on Azure resources and it's in the `azureUtils.ts` file, so it's intended to be used with Azure resources only.  This could probably be refactored to accept more generic resources (not to be confused with `GenericResource` 🤣) though.